tests/Tests/100__deploy/100__installation/102__post_install.robot

```diff
@@ -239,3 +256,11 @@ Verify Errors In Jupyterhub Logs
         ...    container=${pod['spec']['containers'][0]['name']}
         Should Not Contain    ${logs}    ModuleNotFoundError: No module named 'distutils.util'
     END
+
+Check TLS Is Enabled
```

Comment:
Could be renamed to `Verify Grafana Datasources Have TLS Enabled` or similar and be placed in ODS.robot

Reply:
Yes, it is a different test. Since it is a very small change , can we put it in this test?

Reply:
updated